crates/tests/tests/integration.rs

```diff
@@ -173,10 +204,40 @@ async fn test_trbfv_actor() -> Result<()> {
 
     bus.do_send(event);
 
-    // NOTE: We are using node 0 as the aggregator but it is not selected in this seed which is why
-    // there is no CiphernodeSelected event
+    // For score sortition, we need to wait for nodes to process E3Requested and run sortition
+    // Since TicketGenerated is a local-only event (not shared across network), we can't collect it
+    // we need to manually construct the committee that sortition would select
+    println!("Waiting for nodes to process E3Requested...");
+    tokio::time::sleep(Duration::from_millis(200)).await;
```

Comment:
It might be good to test for the events using `take_history_with_timeout()`? That should avoid us needing to add sleeps